File: SOLUTIONS/Code coach problems/Easy/C++/Jungle_Camping.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string a;
	getline(cin, a);
	for(int i = 0; i < a.size(); i++)
	{
		if(i == 0 || a[i] == ' ')
		{
			if(i != 0) i++;
			switch (a[i])
			{
				case 'S':
				cout << "Snake ";
				break;
				case 'G':
				cout << "Lion ";
				break;
				case 'R':
				cout << "Tiger ";
				break;
				case 'C':
				cout << "Bird ";
				break;
			}
		}
	}
	return 0;
}

File: SOLUTIONS/Code coach problems/Easy/Python/Jungle_Camping.py
a = input()
for i in range(len(a)):
	if i == 0 or a[i] == ' ':
		if i != 0:
			i+=1
		if a[i] == 'S':
			print("Snake", end = " ")
		elif a[i] == 'G':
			print("Lion", end = " ")
		elif a[i] == 'R':
			print("Tiger", end = " ")
		elif a[i] == 'C':
			print("Bird", end = " ")
